go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/alkiranet/terraform-provider-alkira
 go 1.16
 
 require (
-	github.com/alkiranet/alkira-client-go v0.8.3
+	github.com/alkiranet/alkira-client-go v0.8.4
```

Comment:
Do we have any rules about versioning the alkira client? I just did a minor version when I added a couple of fields. Looks like you did the same.